docs/build/sdk/swap.md

```diff
@@ -185,28 +185,28 @@ You can use this function to swap a given `assetAmountIn` of the
 `assetIn/assetOut` pair to pool.
 
 ```typescript
-const res = await pool.swapExactAmountIn(
+const res = await pool.swapExactAmountIn({
   signer,
   assetIn,
   assetAmountIn,
   assetOut,
   minAmountOut,
   maxPrice,
-  false
-);
+  callbackOrPaymentInfo: false,
+});
 ```
 
-**Arguments**
+**Object Arguments**
 
-| Name                  | Type                   | Introduction                                                                       |
-| --------------------- | ---------------------- | ---------------------------------------------------------------------------------- |
-| signer                | KeyringPairOrExtSigner | The actual signer provider to sign the transaction.                                |
-| assetIn               | string                 | Asset entering the pool.                                                           |
-| assetAmountIn         | string                 | Amount that will be transferred from the provider to the pool.                     |
-| assetOut              | boolean                | Asset leaving the pool.                                                            |
-| minAmountOut          | string                 | Minimum asset amount that can leave the pool.                                      |
-| maxPrice              | string                 | Market price must be equal or less than the provided value.                        |
-| callbackOrPaymentInfo | boolean                | "true" to get txn fee estimation otherwise callback to capture transaction result. |
+| Name                  | Type                   | Description                                                                       |
+| --------------------- | ---------------------- | --------------------------------------------------------------------------------- |
+| signer                | KeyringPairOrExtSigner | The actual signer provider to sign the transaction                                |
+| assetIn               | string                 | Asset entering the pool                                                           |
+| assetAmountIn         | string                 | Amount that will be transferred from the provider to the pool                     |
```

Comment:
This should probably be a number, as well. I'm also wondering: How do I specify amounts? Are they specified as fixed-point numbers or as floats?

Reply:
These are points I too certainly want to add but I think I might clog the description here. I also think examples are best way of explanation. Any suggestions so that we maintain the format? @Whisker17 @maltekliemann

Reply:
I see, need to come up with something.